src/send_packets.c

```diff
@@ -93,11 +94,37 @@ float2timer (float time, struct timeval *tvp)
     tvp->tv_usec = n * 100000;
 }
 
-/* buffer should be "file_name" */
-int
-parse_play_args (char *buffer, pcap_pkts *pkts)
+static char* find_file(const char* filename)
 {
-    pkts->file = strdup (buffer);
+    if (filename[0] == '/' || !scenario_path) {
+        return strdup(filename);
+    }
+
+    char* fullpath = NULL;
+    if (asprintf(&fullpath, "%s/%s", scenario_path, filename) < 0) {
```

Comment:
Alright, I think I'll just change this so `MAX_PATH` based logic and open another issue about this that can be perused separately.